doc/userguide/unix-socket.rst

```diff
@@ -42,11 +42,13 @@ example to write custom scripts:
 
 Commands in standard running mode
 ---------------------------------
-You may need to install ``suricatasc`` if you have not done so, running the following command from python/suricatasc
+Runnable script for suricatasc is available in `python/bin` directory of suricata. You can
+run it with the following commands.
 
 ::
 
-  sudo python setup.py install
+  cd python
+  sudo ./bin/suricatasc
```

Comment:
These changes are correct, but this does need a rewrite.  More of a mental note for now.